graphsemnet/gsn_api.py

```diff
@@ -0,0 +1,147 @@
+import copy
+import numpy as np
+
+
+class GraphOperator(object):
```

Comment:
Thanks! Looks great. Cold you just add some docstring to the two classes, so that future us will figure out what they do?